runtime/vm/createramclass.cpp

```diff
@@ -2740,6 +2804,17 @@ internalCreateRAMClassFromROMClass(J9VMThread *vmThread, J9ClassLoader *classLoa
 	J9ClassLoader* hostClassLoader = classLoader;
 	J9Module* module = NULL;
 
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+	J9Class **valueTypes = NULL;
+	J9Class *tmpValueTypes[VALUE_TYPE_CLASS_COUNT_ARRAY_THRESHOLD];
+	if (romClass->valueTypeClassCount > VALUE_TYPE_CLASS_COUNT_ARRAY_THRESHOLD) {
+			valueTypes = (J9Class **) malloc(romClass->valueTypeClassCount * sizeof(J9Class *));
+	} else {
```

Comment:
assign `valueTypes` immediately since thats the common case. This way you only need the `if` and and not the `else`